<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QFormLayout" name="qchartForm"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="examplesCnfg">
      <item>
       <widget class="QSpinBox" name="examplenSpin">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadExBtn">
        <property name="text">
         <string>Load Example</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="tabSpacing">
       <number>1</number>
      </property>
      <widget class="QWidget" name="toolBoxPage1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>559</width>
         <height>561</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Interpolation by newton polynom (by value x find y and find root if have)</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="HLfuncInterp">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="nodesCount">
                  <item>
                   <widget class="QLabel" name="NodesLabel">
                    <property name="text">
                     <string>Nodes (min 2)</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="nodesCountSpinB">
                    <property name="minimum">
                     <number>2</number>
                    </property>
                    <property name="maximum">
                     <number>100</number>
                    </property>
                    <property name="singleStep">
                     <number>1</number>
                    </property>
                    <property name="value">
                     <number>10</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QTableWidget" name="nodesTable">
                  <property name="autoScrollMargin">
                   <number>16</number>
                  </property>
                  <property name="rowCount">
                   <number>10</number>
                  </property>
                  <property name="columnCount">
                   <number>2</number>
                  </property>
                  <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                   <bool>false</bool>
                  </attribute>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <column/>
                  <column/>
                  <item row="0" column="0">
                   <property name="text">
                    <string>0.137</string>
                   </property>
                  </item>
                  <item row="0" column="1">
                   <property name="text">
                    <string>0.005</string>
                   </property>
                  </item>
                  <item row="1" column="0">
                   <property name="text">
                    <string>0.343</string>
                   </property>
                  </item>
                  <item row="1" column="1">
                   <property name="text">
                    <string>0.035</string>
                   </property>
                  </item>
                  <item row="2" column="0">
                   <property name="text">
                    <string>0.421</string>
                   </property>
                  </item>
                  <item row="2" column="1">
                   <property name="text">
                    <string>0.055</string>
                   </property>
                  </item>
                  <item row="3" column="0">
                   <property name="text">
                    <string>0.454</string>
                   </property>
                  </item>
                  <item row="3" column="1">
                   <property name="text">
                    <string>0.065</string>
                   </property>
                  </item>
                  <item row="4" column="0">
                   <property name="text">
                    <string>0.626</string>
                   </property>
                  </item>
                  <item row="4" column="1">
                   <property name="text">
                    <string>0.135</string>
                   </property>
                  </item>
                  <item row="5" column="0">
                   <property name="text">
                    <string>0.664</string>
                   </property>
                  </item>
                  <item row="5" column="1">
                   <property name="text">
                    <string>0.155</string>
                   </property>
                  </item>
                  <item row="6" column="0">
                   <property name="text">
                    <string>0.731</string>
                   </property>
                  </item>
                  <item row="6" column="1">
                   <property name="text">
                    <string>0.195</string>
                   </property>
                  </item>
                  <item row="7" column="0">
                   <property name="text">
                    <string>0.843</string>
                   </property>
                  </item>
                  <item row="7" column="1">
                   <property name="text">
                    <string>0.275</string>
                   </property>
                  </item>
                  <item row="8" column="0">
                   <property name="text">
                    <string>0.891</string>
                   </property>
                  </item>
                  <item row="8" column="1">
                   <property name="text">
                    <string>0.315</string>
                   </property>
                  </item>
                  <item row="9" column="0">
                   <property name="text">
                    <string>0.985</string>
                   </property>
                  </item>
                  <item row="9" column="1">
                   <property name="text">
                    <string>0.405</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="ConfigHLay">
                <item>
                 <layout class="QVBoxLayout" name="xTargetVLay">
                  <item>
                   <widget class="QLabel" name="xTargetLabel">
                    <property name="text">
                     <string>target x</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="xTargetSpinB">
                    <property name="decimals">
                     <number>3</number>
                    </property>
                    <property name="minimum">
                     <double>-99.000000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.100000000000000</double>
                    </property>
                    <property name="value">
                     <double>0.600000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="polyPowVLay">
                  <item>
                   <widget class="QLabel" name="polyPowLabel">
                    <property name="text">
                     <string>poly power</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="polyPowSpinB">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="value">
                     <number>4</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="calcBtn">
                <property name="text">
                 <string>Calc</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="resultsVLay" stretch="10,1">
              <item>
               <widget class="QTableView" name="workTable"/>
              </item>
              <item>
               <widget class="QTextBrowser" name="resultTextBrowser"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="toolBoxPage2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>559</width>
         <height>561</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Search for the roots of a system of nonlinear equations</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="nodes2Count">
                <item>
                 <widget class="QLabel" name="NodesLabel_2">
                  <property name="text">
                   <string>Nodes (min 2)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="nodesCountSpinB_2">
                  <property name="minimum">
                   <number>2</number>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="singleStep">
                   <number>1</number>
                  </property>
                  <property name="value">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTableWidget" name="nodes2Table">
                <property name="autoScrollMargin">
                 <number>16</number>
                </property>
                <property name="rowCount">
                 <number>10</number>
                </property>
                <property name="columnCount">
                 <number>2</number>
                </property>
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                 <bool>false</bool>
                </attribute>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <column/>
                <column/>
                <item row="0" column="0">
                 <property name="text">
                  <string>0.1</string>
                 </property>
                </item>
                <item row="0" column="1">
                 <property name="text">
                  <string>-0.284</string>
                 </property>
                </item>
                <item row="1" column="0">
                 <property name="text">
                  <string>0.3</string>
                 </property>
                </item>
                <item row="1" column="1">
                 <property name="text">
                  <string>-0.300</string>
                 </property>
                </item>
                <item row="2" column="0">
                 <property name="text">
                  <string>0.4</string>
                 </property>
                </item>
                <item row="2" column="1">
                 <property name="text">
                  <string>-0.293</string>
                 </property>
                </item>
                <item row="3" column="0">
                 <property name="text">
                  <string>0.48</string>
                 </property>
                </item>
                <item row="3" column="1">
                 <property name="text">
                  <string>-0.266</string>
                 </property>
                </item>
                <item row="4" column="0">
                 <property name="text">
                  <string>0.6</string>
                 </property>
                </item>
                <item row="4" column="1">
                 <property name="text">
                  <string>-0.165</string>
                 </property>
                </item>
                <item row="5" column="0">
                 <property name="text">
                  <string>0.68</string>
                 </property>
                </item>
                <item row="5" column="1">
                 <property name="text">
                  <string>-0.037</string>
                 </property>
                </item>
                <item row="6" column="0">
                 <property name="text">
                  <string>0.75</string>
                 </property>
                </item>
                <item row="6" column="1">
                 <property name="text">
                  <string>0.128</string>
                 </property>
                </item>
                <item row="7" column="0">
                 <property name="text">
                  <string>0.88</string>
                 </property>
                </item>
                <item row="7" column="1">
                 <property name="text">
                  <string>0.610</string>
                 </property>
                </item>
                <item row="8" column="0">
                 <property name="text">
                  <string>0.95</string>
                 </property>
                </item>
                <item row="8" column="1">
                 <property name="text">
                  <string>0.988</string>
                 </property>
                </item>
                <item row="9" column="0">
                 <property name="text">
                  <string>1.03</string>
                 </property>
                </item>
                <item row="9" column="1">
                 <property name="text">
                  <string>1.547</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="ConfigHLay_2">
              <item>
               <layout class="QVBoxLayout" name="xTargetVLay_2">
                <item>
                 <widget class="QLabel" name="xTargetLabel_2">
                  <property name="text">
                   <string>aux polypow</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="sysauxtSpinB">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="value">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="polyPowVLay_2">
                <item>
                 <widget class="QLabel" name="polyPowLabel_2">
                  <property name="text">
                   <string>root polypow</string>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="sysrootPolyPowSpinB">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="value">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="calcsysequarootBtn">
              <property name="text">
               <string>Calc system equetion root</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="5,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QTableView" name="tableAuxView"/>
              </item>
              <item>
               <widget class="QTableView" name="tableResView"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTextBrowser" name="resultTextBrowser_2"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
